service/src/main/kotlin/fi/espoo/evaka/attachment/AttachmentsController.kt

```diff
@@ -265,19 +277,18 @@ class AttachmentsController(
         documentClient.delete(filesBucket, "$attachmentId")
     }
 
-    val defaultAllowedAttachmentContentTypes: List<String> = listOf(
-        "image/jpeg",
-        "image/png",
-        "application/pdf",
-        "application/msword",
-        "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
-        "application/vnd.oasis.opendocument.text",
-        "application/x-tika-msoffice",
-        "application/x-tika-ooxml"
-    )
+    private val defaultAllowedAttachmentContentTypes: Set<ContentType> =
+        setOf(
+            ContentType.JPEG,
+            ContentType.PNG,
+            ContentType.PDF,
+            ContentType.MSWORD,
+            ContentType.MSWORD_DOCX,
+            ContentType.OPEN_DOCUMENT_TEXT,
+            ContentType.TIKA_MSOFFICE,
+            ContentType.TIKA_OOXML,
+        )
 
-    val pedagogicalDocumentAllowedAttacmentContentTypes = defaultAllowedAttachmentContentTypes + listOf(
-        "video/*",
```

Comment:
The explicit list of allowed video and audio formats is more restricted than this, but I agree it is better to have those listed out and then updated when real need arises